stream-chat-android/src/main/java/com/getstream/sdk/chat/view/MessageListViewStyle.kt

```diff
@@ -0,0 +1,618 @@
+package com.getstream.sdk.chat.view
+
+import android.content.Context
+import android.graphics.Color
+import android.graphics.Typeface
+import android.util.AttributeSet
+import androidx.core.content.ContextCompat
+import com.getstream.sdk.chat.R
+import com.getstream.sdk.chat.style.TextStyle
+import com.getstream.sdk.chat.view.messages.AvatarStyle
+
+public class MessageListViewStyle(c: Context, attrs: AttributeSet?) {
```

Comment:
Some of the `...Style` classes could not be made `internal` because either they are still used in Java views or are passed as args in public functions.

Reply:
This also means that we definitely broke some of our public API with the changes here. If someone's using custom ViewHolders with either the attachment or message ViewHolder factory, this will mean syntax changes for them. While we probably don't want to list all the changes, we should add something about it into the changelog.